### PR TITLE
[BUGFIX] Correcting instructions for Contributor Package installation

### PR DESCRIPTION
**Pull request by alexsherstinsky**
### Scope
I tried the previous way, and got the message that the project does not exist; however, the suggested way worked.

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted

    ```
    black .

    ruff . --fix
    ```
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: CONTRIBUTING_PACKAGES.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Before you develop your custom Great Expectations package, notify the Great Expe
 
 ## Install the Great Expectations CLI tool
 
-Great Expectations provides the `great_expectations_contrib` command-line interface (CLI) tool to simplify the package contribution process and help you adhere to code base best practices. 
+Great Expectations provides the `great_expectations_contrib` command-line interface (CLI) tool to simplify the package contribution process and help you adhere to code base best practices.
```
Comment (alexsherstinsky) [inline CONTRIBUTING_PACKAGES.md:43]: It is not clear why this text appears as changed -- it seems identical to what was there before, save for the blank space at the end (result of linter).

File: CONTRIBUTING_PACKAGES.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ While you're developing your package, you must use this tool to ensure that it m
 
 ## Initialize a project
 
-After you've enabled the `great_expectations_contrib` CLI tool, you need to initialize an empty package. 
+After you've enabled the `great_expectations_contrib` CLI tool, you need to initialize an empty package.
```
Comment (alexsherstinsky) [inline CONTRIBUTING_PACKAGES.md:75]: It is not clear why this text appears as changed -- it seems identical to what was there before, save for the blank space at the end (result of linter).

File: CONTRIBUTING_PACKAGES.md [hunk 3]
```diff
@@ -150,6 +150,6 @@ Additionally, Great Expectations adds the following tags to indicate issue statu
 
 - The`help wanted` tag identifies useful issues that require help from community contributors to accelerate development.
 
-- The `enhacement` and `expectation-request` tags identify new Great Expectations features that require additional investigation and discussion. 
+- The `enhacement` and `expectation-request` tags identify new Great Expectations features that require additional investigation and discussion.
```
Comment (alexsherstinsky) [inline CONTRIBUTING_PACKAGES.md:153]: It is not clear why this text appears as changed -- it seems identical to what was there before, save for the blank space at the end (result of linter).

File: CONTRIBUTING_PACKAGES.md (hunk 1 above)
Comment (cdkini) [reply CONTRIBUTING_PACKAGES.md:43]: Yup it's a linter rule to clean up excess whitespace

**Review by cdkini — APPROVED** [review]

**Pull request merged**